source/storing-data/archive-versions.rst

```diff
@@ -0,0 +1,226 @@
+as they allow archive versions to be checked,
+and archives with different formats to be dispatched to the appropriate version-specific tools.
+
+All QIIME 2 Archives have:
+
+- a directory named with the Archive UUID, directly under the archive root.
+- a file named "VERSION" within that directory, formatted as shown below
+
+The Archive file system must look like this:
+
+.. figure:: ../img/archive_versions/format_agnostic_archive_structure.svg
+   :alt: Box and Arrow diagram of the guaranteed components of an archive, as described above.
+
+The VERSION file must look like this::
+
+    QIIME 2
+    archive: <archive version>
+    framework: <framework version>
+
+.. note::
+   This file is itentionally not YAML/INI/An actual format. This is to
+   discourage the situation where the format changes from something like YAML to
+   another format and VERSION is updated with it "for consistency".
+
+Version 0
+---------
+
+The original QIIME 2 Archive format, there aren't many V0 Archives "in the wild".
+V0 Archives, in place through 10/24/16, were produced only during QIIME 2's Alpha release,
+prior to version 2.0.6.
+
+- Like all QIIME 2 Archives, there is a directory named with the Archive UUID under the archive root, containing a VERSION file. (This is *not* format-defined. See above.)
+- :term:`Result` data is written to a ``data`` directory, located within this UUID directory.
+- Result UUID, Semantic Type, and Format go in a ``metadata.yaml`` in the same place.
+- The ArchiveFormat class in `v0.py <https://github.com/qiime2/qiime2/blob/master/qiime2/core/archive/format/v0.py>`_ offers convenience methods for loading and parsing ``metadata.yaml`` files.
+
+V0 Archives look like this:
+
+.. figure:: ../img/archive_versions/v0_archive_fmt.svg
+   :alt: Box and Arrow diagram of a v0 archive, as described above.
+
+Version 1
+---------
+
+Created in `PR #171 <https://github.com/qiime2/qiime2/pull/171>`_,
+and first released in QIIME 2 `version 2.0.6 <https://github.com/qiime2/qiime2/releases/tag/2.0.6>`_,
+Version 1 Archives introduce decentralized provenance tracking to QIIME 2.
+`ArchiveFormat V1 <https://github.com/qiime2/qiime2/blob/master/qiime2/core/archive/format/v1.py>`_ inherits all traits of v0,
+modifying its ``__init__()`` and ``write()`` methods only to add provenance capture.
+
+.. note::
+   All ArchiveFormat versions subclass their predecessor.
+
+   E.g. the ArchiveFormat in ``v1.py`` inherits from the ArchiveFormat in ``v0.py``,
+   etc. This improves the interpretability of the version history.
+
+- A ``provenance`` folder is written in the top-level UUID directory
+- ``metadata.yaml``, ``action.yaml`` and ``VERSION`` files are captured for the current Result and all ancestors.
+- The ``action.yaml`` file and associated data artifacts (e.g. sample metadata) are stored in an ``action`` directory.
+- For the current Result, all of this captured data lives directly inside ``provenance``
```

Comment:
I'm not sure if the itemized list is helpful here, especially when you have bullets that "depend" on the prior bullets. On this line, its not clear to me what "this captured data" is referring to.